apps/frontend/src/modules/surveys/components/studio/create/editor-panel/DesignEditor/components/PresetThemes.tsx

```diff
@@ -1,4 +1,5 @@
 import { useThemes } from "@/modules/projects/hooks/useTheme";
```

Comment:
@kaf-lamed-beyt We should probably rethink the implementation of theme. Tight-coupling it with survey doesn't seem like the right way to go. Remember, it's supposed to be under project which means it will still exist even if Survey goes away or if we have other in-app content like `Poll`, `Quick`, `Push` e.t.c. We should probably move it to project module, then utilise event handlers to update things in other modules. Let me know if you need more explanations.

Reply:
Makes sense! I left a message for you BTW.